lms/static/sass/elements/_controls.scss

```diff
@@ -404,7 +404,7 @@
     }
   }
 
-  .btn-link {
+  .btn-secondary {
```

Comment:
Thanks for doing that. (btn-link --> Leave Team)

I would extend %btn-pl-secondary-base (which is in this _controls.scss file) for Edit Team, as that button can have padding / spacing around it.  That extend should map onto our secondary styling as shown in the image below.

![image](https://cloud.githubusercontent.com/assets/2023680/9450504/ed5f88c8-4a76-11e5-9358-cab328a6267b.png)